### DISCUSSION TITLE
模型选项 | OhMyLive2D

### DISCUSSION THREAD

**Discussion by giscus** [oh-my-live2d/oh-my-live2d#37] [Announcements]
# 模型选项 | OhMyLive2D

支持所有Live2D模型版本的Web组件, 快速为您的个人网站添加看板娘.

https://oml2d.com/options/ModelOptions.html

**Comment by elioxu** [comment]
有右对齐的选项吗

**Reply by hacxy** [reply]
是指组件整体靠屏幕右边吗?
可以看下这个选项: <https://oml2d.com/options/Options.html#dockedposition>

**Reply by elioxu** [reply]
是的，非常感谢

**Reply by elioxu** [reply]
我对希望整个组件的位置微调整，以像素为单位移动，有此选项吗
